### DISCUSSION TITLE
when you import maps from stable, is the "date added" and "last played" same as stable?

### DISCUSSION THREAD

**Discussion by NafeeJ** [ppy/osu#25296] [Q&A]
title. i dont really want to import until these are accurate as in the future i would like to see what my song select looked at different points in time.

**Comment by peppy** [comment]
"Date added" will be correct. "Last played" will not be so correct (but will improve via https://github.com/ppy/osu/pull/25303).

**Reply by Chamqp** [reply]
i am still having a issue with lazer not sorting correctly for date added

**Reply by EXtremeExploit** [reply]
Same, and its because date added is per set not per difficulty of a set, i think it has to change for stable parity in my opinion, its really annoying having some of my maps in completely different places than where i expect
